README.md

```diff
@@ -0,0 +1,3 @@
+This script generates a password using your first/last name and random special chars. The output should be your first name and some random chars, you can specify the length of your pw.
```

Comment:
we should make the README.md much more fancier, with proper headings, maybe drop a GIF in it for comedic purposes, stylize the README so people should actually want to read the README.